src/main/java/net/minestom/server/entity/metadata/animal/CamelMeta.java

```diff
@@ -0,0 +1,30 @@
+package net.minestom.server.entity.metadata.animal;
+
+import net.minestom.server.entity.Entity;
+import net.minestom.server.entity.Metadata;
+import org.jetbrains.annotations.NotNull;
+
+public class CamelMeta extends AbstractHorseMeta {
+    public static final byte OFFSET = AbstractHorseMeta.MAX_OFFSET;
+    public static final byte MAX_OFFSET = OFFSET + 1;
```

Comment:
Pretty sure this is supposed to be 1 above the max although I'm not sure as it appears pretty inconsistent.

E.g:
  - BatMeta has 1 field and is +0 (same concept as this)
  - AgeableMobMeta has 1 field and is +1 (so 1 above the max index it uses)
  - FireballMeta has 0 fields and is +0 (so same theory as max + 1, otherwise it would have to be `MAX_OFFSET = OFFSET - 1` which seems obviously wrong)

Basically I have no idea what this should be. Maybe a separate PR later to make them all consistent.

Reply:
Confirmed, it should be 1 above the max so `public static final byte MAX_OFFSET = OFFSET + 2;`